dev/Dockerfile

```diff
@@ -4,4 +4,6 @@ RUN curl -L -o cincinnati-graph-data.tar.gz https://api.openshift.com/api/upgrad
 
 RUN mkdir -p /var/lib/cincinnati-graph-data && tar xvzf cincinnati-graph-data.tar.gz -C /var/lib/cincinnati-graph-data/ --no-overwrite-dir --no-same-owner
 
-CMD ["/bin/bash", "-c" ,"exec cp -rp /var/lib/cincinnati-graph-data/* /var/lib/cincinnati/graph-data"]
+RUN mkdir -p /var/lib/cincinnati-graph-data/signatures/sha256/beda83fb057e328d6f94f8415382350ca3ddf99bb9094e262184e0f127810ce0 && curl -L https://mirror.openshift.com/pub/openshift-v4/signatures/openshift/release/sha256=beda83fb057e328d6f94f8415382350ca3ddf99bb9094e262184e0f127810ce0/signature-1 >/var/lib/cincinnati-graph-data/signatures/sha256/beda83fb057e328d6f94f8415382350ca3ddf99bb9094e262184e0f127810ce0/signature-1 && echo 1.1.0 >/var/lib/cincinnati-graph-data/version
```

Comment:
Waiting for openshift/cincinnati#876 to merge to be able to use 1.2.0 in `version` here.

Reply:
openshift/cincinnati#876 merged, so I'm back to using 1.2.0 here.